webpack.common.config.ts

```diff
@@ -187,6 +189,23 @@ module.exports = {
       minify: htmlMinifyConfig,
       inject: false
     }),
+    new HtmlWebpackPlugin({
+      chunks: ['blog_admin'],
+      filename: 'blog-admin-page.mainpage.html',
+      hybrid: true,
```

Comment:
Is hybrid required? Any component/ service in this page that uses angularjs?

Reply:
not required. Changed to false.